src/testing/cluster/storage_checker.zig

```diff
@@ -73,23 +69,34 @@ pub const StorageChecker = struct {
         var free_set = try vsr.FreeSet.init(allocator, Storage.grid_blocks_max);
         errdefer free_set.deinit(allocator);
 
+        var client_sessions = try vsr.ClientSessions.init(allocator);
+        errdefer client_sessions.deinit(allocator);
+
         var free_set_buffer = try allocator.alignedAlloc(
             u8,
             @alignOf(u64),
             vsr.FreeSet.encode_size_max(Storage.grid_blocks_max),
         );
-        errdefer allocator.free(free_set);
+        errdefer allocator.free(free_set_buffer);
```

Comment:
Typo. Not found by the compiler (prior to adding `client_sessions_buffer`) since the `errdefer` was not previously reachable (nothing could possibly error before here and the end of the function).